firebase-database/src/main/java/com/google/firebase/database/Query.java

```diff
@@ -478,6 +478,96 @@ private Query startAt(Node node, String key) {
     return new Query(repo, path, newParams, orderByCalled);
   }
 
+  /**
+   * Create a query constrained to only return child nodes with a value greater than the given
+   * value, using the given orderBy directive or priority as default.
```

Comment:
I just realized that all these comments say "value greater than", when it should be "value less than".